guides/README.md

```diff
@@ -67,6 +67,22 @@ It's also possible to check links; the following command will check all links ex
 
 	make linkchecker
 
+## Building Locally Using a Container
+
+You can build Foreman documentation locally using a container image.
+This requires the cloned git repository plus an application such as Podman or Docker to build and run container images.
+
+1. Build container image:
+
+       sudo podman build --no-cache --file Containerfile --tag foreman_documentation:1.0 .
```

Comment:
Why `no-cache`?

On Fedora 34, this also works for me:
```suggestion
       podman build --tag foreman_documentation:1.0 .
```

In fact, the that this build is with sudo and the one below is without is guaranteed to give problems. You must run either both as root or both as a normal user.

Reply:
Fixed.